Packs/Core/Integrations/CortexCoreIR/CortexCoreIR.py

```diff
@@ -69,6 +81,59 @@ def report_incorrect_wildfire_command(client: Client, args) -> CommandResults:
     )
 
 
+def handle_prevalence_command(client: Client, command: str, args: dict):
+    key_names_in_response = {
+        'ip': 'ip_address',
+        'domain': 'domain_name',
+        'process': 'process_name',
+        'cmd': 'process_command_line',
+        'hash': 'sha256',
+        'registry': 'key_name'
+    }
+    args.pop('integration_context_brand', None)
+    args.pop('integration_name', None)
+    if command == 'core-get-registry-analytics-prevalence':
+        # arg list should in the following structure:
+        #   args: [
+        #       {"key_name": "some_key1", "value_name": "some_value1"},
+        #       {"key_name": "some_key2", "value_name": "some_value2"}
+        #       ]
+
+        args_list = []
+        keys = argToList(args.get('key_name'))
+        values = argToList(args.get('value_name'))
+        if len(keys) != len(values):
+            raise DemistoException('Number of elements in key_name argument should be equal to the number '
+                                   'of elements in value_name argument.')
+        for i in range(len(keys)):
+            args_list.append({'key_name': keys[i], 'value_name': values[i]})
```

Comment:
This is python, not C.
```suggestion
        for key, value in zip(keys, values):
            args_list.append({'key_name': key, 'value_name': value})
```